### DISCUSSION TITLE
New expand control, that expand horizontally, to show details

### DISCUSSION THREAD

**Discussion by eduardoagr** [CommunityToolkit/Maui#1835] [New Feature Discussions]
I needed a way to expand horizontally, and I was looking into our toolkit, but the expander doesn't offer an option to do that.

And my app, that is like an app meeting needs something like this.

So, I though am a good idea to bring it here (I know is a very stupid and simple control)

and this could also take care of this. 

https://github.com/dotnet/maui/issues/7345

https://reccloud.com/u/zjcue1t

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.
